module/functions/azure/azcli/Invoke-AzCliRestCommand.ps1

```diff
@@ -50,7 +50,7 @@ function Invoke-AzCliRestCommand
         [string] $Method = "GET",
         
         [Parameter(ParameterSetName = 'Body as hashtable')]
-        [hashtable] $Body,
+        $Body,
```

Comment:
While this fixes the problem in which you weren't allowed to pass an array of hashtables, do we now suffer slightly from a lack of input validation?

Should we add something that checks that this is either a hashtable or an array of hashtables, and report an error if it's neither? Or will a moderately comprehensible error emerge in that case without any explicit handling?

Reply:
You would get a JSON parsing error, so 'moderately comprehensible' might actually be the best we can hope for!  I'll add something.